relayer/chains/wasm/types/types.go

```diff
@@ -19,13 +19,20 @@ type TxSearchParam struct {
 }
 
 func (param *TxSearchParam) BuildQuery() string {
-	startHeight := &Query{
-		Field: "tx.height", Value: param.StartHeight,
-		Operator: QueryOperator.Gte,
-	}
-	endHeight := &Query{
-		Field: "tx.height", Value: param.EndHeight,
-		Operator: QueryOperator.Lte,
+	var queries []QueryExpression
+
+	if param.StartHeight-param.EndHeight < 2 { // if the difference is less than 2, it means only one block
```

Comment:
This expression is always negative. Also I think the correct expression is to check if startHeight is equal to EndHeight. This means that we have to query only 1 height. If the difference is less than 2 does not mean we query one block.

Reply:
https://github.com/icon-project/centralized-relay/pull/302/commits/62d15f0373e5b0203fe0e25c5677cd03f1533f04